### ISSUE TITLE
Spike: Investigate the possibility of MFA in Cognito in the short term

### ISSUE THREAD

**Issue by willmcvay** [reapit/foundations#8091]
**Background context or User story:**

_While we are working on the long term SSO milestones, is it possible / feasible to deliver a short term MFA win with the existing Cognito SSO?_

**Comment by plittlewood-rpt** [comment]
Agreed to move on this based on simple POC result. Additional tickets logged to current sprint
